src/inmanta/agent/agent.py

```diff
@@ -774,13 +776,15 @@ async def do_run_dryrun(self, version: int, dry_run_id: uuid.UUID) -> None:
                             resource.rvid,
                             undeployable[resource.rvid],
                         )
+                        # TODO: SKIP IS NO CHANGE????
```

Comment:
I'm in favor of changing this, but perhaps this is something for another ticket?

Reply:
https://github.com/inmanta/inmanta-core/issues/7588